infrastructure/deploy.sh

```diff
@@ -402,7 +402,7 @@ container_running=$(ssh -o StrictHostKeyChecking=no \
                         -o ServerAliveInterval=15 \
                         -o ConnectTimeout=5 \
                         -i data-refinery-key.pem \
-                        "ubuntu@$API_IP_ADDRESS"  "docker ps -a" | grep dr_api || echo "")
+                        "ubuntu@$API_IP_ADDRESS"  "docker ps -a" 2> /dev/null | grep dr_api || echo "")
```

Comment:
confirm that you definitely want this change included

Reply:
Definitely! When the API cylces (which is often) this will output an error about not being able to connect right at the end of the deploy that makes a successful deploy seem less obviously successful.